reflex/components/plotly/plotly.py

```diff
@@ -4,14 +4,18 @@
 from typing import Any, Dict, List
 
 from reflex.base import Base
-from reflex.components.component import NoSSRComponent
+from reflex.components.component import Component, NoSSRComponent
+from reflex.components.core.cond import color_mode_cond
 from reflex.event import EventHandler
 from reflex.vars import Var
 
 try:
-    from plotly.graph_objects import Figure
+    from plotly.graph_objects import Figure, layout
+
+    Template = layout.Template
 except ImportError:
```

Comment:
Do we still need to handle this import error now that we have lazy loading for the imports?
We should just raise and tell them to install plotly package

Reply:
added a warning if users try

```python
>>> import reflex as rx
>>> rx.plotly
Warning: Plotly is not installed. Please run `pip install plotly`.
<bound method Plotly.create of <class 'reflex.components.plotly.plotly.Plotly'>>
```

it does still work though